### DISCUSSION TITLE
Is there a way to use the `ordering` option in pynetbox?

### DISCUSSION THREAD

**Discussion by moises-perez-tfs** [netbox-community/pynetbox#556] [Q&A]
NetBox API seem to support an `ordering` option (_e.g._ the `/ipam/prefixes/` endpoint supports it), but I did not find it in pynetbox. Is there anything in pynetbox to make it work?

**Comment by variable** [comment]
After inspecting the code, you can add the `ordering` into `filter()` like `filter(ordering='id')`

**Reply by achmadputrak** [reply]
hi, how about descending? thank you

**Reply by variable** [reply]
have you tried `filter(ordering='-id')`?

**Reply by achmadputrak** [reply]
it works now, thankyou
